assembler.py

```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
             else:
                 if source[0].lower() == 'org':
                     # Indicates at what memory location it will begin storing instructions
+                    if len(source) > 2:
+                        # there is more than one value after the 'org' - invalid address.
+                        print('org initialization failed: too many arguments after org.')
+                        break
+
                     org_address = source[1]
+                    try:
+                        value = int(org_address, 16)
+                    except ValueError:
+                        print('given hexadecimal value is invalid.')
+                        break
```

Comment:
Add ValueError to main.py try-except block

Use this instead `org_address = int(source[1], 16)`